site/docs/v1/tech/installation-guide/monitor.adoc

```diff
@@ -79,3 +79,16 @@ You can also enable JMX metrics as outlined in the link:/docs/v1/tech/troublesho
 ==== Prometheus Endpoint
 
 The default system metrics are also available in a Prometheus compatible form. link:/docs/v1/tech/tutorials/prometheus[This tutorial explains how to set up Prometheus to scrape the FusionAuth metrics].
+
+=== Load Testing
+
+You can load test FusionAuth to determine if its performance will meet your calculated needs. The FusionAuth team tests the performance of FusionAuth regularly, but every situation is different and running your own load tests on your instance provides useful information.
+
+There is an https://github.com/FusionAuth/fusionauth-load-tests/[open source project] which provides some useful scripts for load testing. Scenarios currently supported include:
+
+* Creating tenants
+* Creating applications
+* Registering users for applications
+* Logging users in using the Login API
+* Beginning the Authorization Code grant
+
```

Comment:
I have mixed feelings about pointing to our own load test framework. It is not particular documented, and is very purpose built. It may be better to suggest a third party load test framework? 

We could start here and just say use at your own risk or something like that.

Reply:
Fair.